src/libsyntax/feature_gate.rs

```diff
@@ -674,6 +671,9 @@ declare_features! (
     // Allows all literals in attribute lists and values of key-value pairs.
     (accepted, attr_literals, "1.30.0", Some(34981), None),
     (accepted, panic_handler, "1.30.0", Some(44489), None),
+    // Used to preserve symbols (see llvm.used)
+    (accepted, used, "1.29.0", Some(40289), None),
```

Comment:
Time flies, this should be 1.30.0 now (or 1.31.0 if it can't be merged this week).